docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -30,6 +31,40 @@ All modules must have the following sections defined in this order:
 
 
 
+Copyright
+----------------------
+
+The first 3 lines of every module should look the same. The first line is the
+shebang, but the next 2 lines are the important bit: copyright statement and
+license.
+
+.. code-block:: python
+
+    #!/usr/bin/python
+    # Copyright © 2017 [REPLACE THIS]
+    # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+Every file with a copyright header must include the original author. Major
+additions to the module are allowed to add an additional copyright line. This
+is especially true of rewrites, but original authorship copyright messages
+should be preserved. If applicable, contributors may have their rights assigned
+to their employer for example: "Copyright © 2017 Red Hat, Inc."
```

Comment:
What does "Code from Ansible" mean here? Is that "Ansible, the professional team that gets paid to write Ansible code" or "Code from Ansible contributors"? Maybe "The Red Hat/Ansible team is standardising ....." would be better?